docs/section21/Rule21-7.md

```diff
@@ -0,0 +1,38 @@
+
+# Boiler - Rule 21-7  
+
+**Rule ID:** 21-7  
+**Rule Description:** When baseline building requires boilers, systems 1,5,7,11 and 12 - Model HWST = 180F and return design temp = 130F.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.3 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12
+2. B-RMR is not modeled with purchased heating.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12: `if PLACEHOLDER_AIRSIDE_SYSTEM_RULE-X-XX:`
+2. B-RMR is not modeled with purchased heating: `if NOT Rule-21-1:`
+
```

Comment:
the second applicability check is confusing for me. does this mean 21-1 is not applicable or is fail? also, since we are not planning on implementing 21-1, we should probably have some other applicability check to verify that there is no purchased heating.
@mkarpman any thoughts?

Reply:
Since we have several mandatory rules that only apply to projects that do not have purschased heating, I think Rule 21-1 must be mandatory. Then the applicability check would be that Rule 21-1 was checked and the outcome of this check is that project does not have purchased heating.

Reply:
I think 21-18 (that there is a single hot water plant) and 21-11 (that there should only be one primary loop) should also be added as applicability checks.

Reply:
Added 21-1, 21-18 and 21-11 as applicability check.

Reply:
I looked again at the applicability diagram that we have in the spreadsheet [https://onedrive.live.com/edit.aspx?cid=796769604e905084&page=view&resid=796769604E905084!170&parId=796769604E905084!156&authkey=!ACtFTkgcGDgG1Qk&app=Excel](url), and think that based on our implementation of 21-1 (to only check if proposed design has purchased heating), we need to also add check that the baseline system is of type 1,5,7,11 or 12.